lib/kafo/hook_context.rb

```diff
@@ -61,6 +61,13 @@ def add_module(module_name, mapping = nil)
       self.kafo.add_module(module_name)
     end
 
+    # Check if a module is enabled in the current configuration.
+    # examples:
+    #   module_enabled?('example')
+    def module_enabled?(module_name)
+      self.kafo.module(module_name).enabled?
```

Comment:
oh, I forgot module can change the value of `@enabled`, from looking at the `enabled?` method it seems it's only memoization of  `Configuration#module_enabled?`. ok :+1: for this change then.